cmd/integration_link.go

```diff
@@ -447,6 +492,18 @@ func interactiveCreate() (scalingo.SCMRepoLinkCreateParams, error) {
 		hoursBeforeDestroyOnStale := uint(hoursBeforeDestroyOnStale64)
 		params.HoursBeforeDeleteStale = &hoursBeforeDestroyOnStale
 	}
+
+	forksAllowed := false
+	err = survey.AskOne(&survey.Confirm{
+		Message: "Allow review apps to be created from forks:",
+		Help:    "Only allow automatic review apps deployments from forks if you trust the owners of those forks, as this could lead to security issues",
+		Default: forksAllowed,
+	}, &forksAllowed, nil)
+	if err != nil {
+		return params, err
```

Comment:
I think we try to always wrap errors into our own, using `errgo.Notef` for example (errgo when our custom error library is not available).

But I see here that there are other places like you did. I don't know if that was actually on purpose or not

Reply:
Well, in order to be consistent, I made this logic on all the integration_link.go file